src/test/java/org/opensearch/agent/tools/RAGToolTests.java

```diff
@@ -261,17 +291,6 @@ public void testRunWithEmptyInput() {
         ragTool.run(Map.of(INPUT_FIELD, ""), listener);
     }
 
-    @Test
-    public void testRunWithMalformedInput() throws IOException {
```

Comment:
Why remove this test case?

Reply:
When taking a string from the input,  it no longer checks json format. It also gives more flexibility for user questions.